app/screens/global_drafts/index.tsx

```diff
@@ -87,13 +164,28 @@ const GlobalDrafts = ({componentId}: Props) => {
             </View>
             {!switchingTeam &&
             <View style={containerStyle}>
-                <GlobalDraftsList
-                    location={Screens.GLOBAL_DRAFTS}
-                />
+                {
+                    scheduledPostsEnabled ? (
+                        <TabbedContents
+                            initialTab={initialTab || DRAFT_SCREEN_TAB_DRAFTS}
+                            draftsCount={draftsCount}
+                            scheduledPostCount={scheduledPostCount}
+                            drafts={draftList}
+                            scheduledPosts={scheduledPostList}
+                        />
+                    ) : draftList
+                }
             </View>
             }
         </SafeAreaView>
     );
 };
 
-export default GlobalDrafts;
+const enhanced = withObservables([], ({database}: WithDatabaseArgs) => {
+    const scheduledPostsEnabled = observeConfigBooleanValue(database, 'ScheduledPosts');
+    return {
+        scheduledPostsEnabled,
+    };
+});
+
+export default withDatabase(enhanced(GlobalDraftsAndScheduledPosts));
```

Comment:
I prefer this be in its own file

Reply:
This is causing a merge conflict for the next PR. I'll make the change once all PRs are merged into the feature branch.

I've created a JIRA ticket for this: [MM-63437](https://mattermost.atlassian.net/browse/MM-63437).